test/run

```diff
@@ -50,7 +50,7 @@ main() {
       echo "$indent$name"
       continue
     elif [ -x enabled ] && ! ./enabled; then
-      echo "$indent$name (disabled)" | colorize $yellow $none
+      printf "${yellow}$indent$name (disabled)${none}\n"
```

Comment:
`$name` could be pretty arbitrary as it comes from the filesystem, so we should probably not put it in the `printf` format string.